src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -422,9 +441,47 @@ def build_purpose_or_feature_section_and_update_vendor_map(
                 privacy_declaration_row=privacy_declaration_row,
             )
 
+        # Separately, add AC Vendor to Vendor Consent Map if applicable
+        add_ac_vendor_to_vendor_consent_map(
+            vendor_map=vendor_map,
+            tcf_vendor_component_type=tcf_vendor_component_type,
+            privacy_declaration_row=privacy_declaration_row,
+        )
+
     return non_vendor_record_map, vendor_map
 
 
+def add_ac_vendor_to_vendor_consent_map(
+    vendor_map: Dict[str, VendorRecord],
+    tcf_vendor_component_type: VendorSectionType,
+    privacy_declaration_row: Row,
+) -> None:
+    """
+    Add systems with ac.*-prefixed vendor records to the Vendor Consent section if applicable.
+
+    FE shows Consent toggle only for AC vendors, and they are not required to have Privacy Declarations
+    """
+    vendor_id, system_identifier = get_system_identifiers(privacy_declaration_row)
+
+    if not tcf_vendor_component_type == TCFVendorConsentRecord:
+        return
+
+    if not system_identifier.startswith(AC_PREFIX):
+        return
+
+    if system_identifier in vendor_map:
+        return
```

Comment:
out of curiosity - do you think this reads easier as 3 separate conditionals rather than joining them in a single conditional with `or`s? just caught my eye since it's not the way i would have written it, but i'm guessing this was deliberate so i'm curious to hear! no need to adjust obviously.

Reply:
I do think it's easier to parse this way 😎